src/main/java/com/sda/zdjavapol107/statki/ship/ShipService.java

```diff
@@ -0,0 +1,261 @@
+package com.sda.zdjavapol107.statki.ship;
+
+import com.sda.zdjavapol107.statki.MessagesPrinter;
+import com.sda.zdjavapol107.statki.model.PointShipsGame;
+
+import java.util.ArrayList;
+import java.util.InputMismatchException;
+import java.util.List;
+import java.util.Scanner;
+
+public class ShipService {
+
+    private ShipDirection shipDirectionEnum;
+    private boolean isShipPositionOk = false;
+    private int choiceShipSize = 1;
+    protected int fourSizeShip = 1;
+    protected int threeSizeShip = 2;
+    protected int twoSizeShip = 3;
+    protected int oneSizeShip = 4;
+    public int shipDirection;
```

Comment:
widze tutaj zmienne shipDirection, isShipPositionOk <- to sa zmienne dotyczace jednego konkretnego statku :D
mysle ze powinniscie oprocz klasy ship service ktora zarzadza statkami pojsc w strone zeby zrobic tez klasa reprezentujaca statek. 
podpowiedz ladnie mozecie to ograc poprzez zrobienie interfejsu ship i potem 4 implementacji dla kazdego rodzaju statku. 
Z takim podejsciem bedziecie mogli stowrzyc List<Ship> na ktorym bedziecie wykonywac wszystkie operacje :D 

Sprobujcie podejsc do tego bardziej obiektowo i generycznie bo strach pomyslec co by bylo gdyby na planszy bylo 50 rodzajow statkow :D